### ISSUE TITLE
dangling line in reaction metadata function

### ISSUE THREAD

**Issue by isaacsas** [SciML/Catalyst.jl#1018]
@TorkelE is this a typo?

https://github.com/SciML/Catalyst.jl/blob/36c10b8661f44054571a7a70361ed7fd1315189f/src/reaction.jl#L503

**Comment by isaacsas** [comment]
Or maybe this is a bad format? But in that case can we split it so it is clear what is going on?

**Comment by TorkelE** [comment]
Yeah, it's a bad format. There are some like that I tried like 5 times to get the formatter to do it properly, but it wouldn't have it.
